code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -1,3 +1,9 @@
+/obj/item/ammo_casing/a556x45
+	desc = "A 5.56x45 bullet casing."
+	caliber = "556x45"

```

Comment:
Right, maybe when I make a follow-up PR SOON(TM) to add new guns, ammo types and streamline projectiles to actually use the shot caliber